src/test/java/jalau/at18/katas/helloworld/joseg/MainTest.java

```diff
@@ -9,5 +9,6 @@ public class MainTest {
     public void mainShouldSayHello() {
         Main main = new Main();
         assertEquals("Hello wooooooorld!!!", main.sayHello());
+        // Se puede tener mas de un assert
```

Comment:
I think this was one of your notes 😄 be careful when commiting these changes, before running git add, it is a good idea to run a git diff so these are detected before commiting unexpected changes